Description: Point to netx.jar to provide the JNLP classes
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -148,7 +148,7 @@
             <artifactId>jnlp</artifactId>
             <version>1.6</version>
             <scope>system</scope>
-            <systemPath>${java.home}/lib/javaws.jar</systemPath>
+            <systemPath>/usr/share/java/netx.jar</systemPath>
         </dependency>
     </dependencies>
     <reporting>
@@ -207,4 +207,4 @@
             <url>http://download.java.net/maven/2/</url>
         </repository>
     </repositories>
-</project>
\ No newline at end of file
+</project>
